core/network/helpers/stream_proxy.hpp

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_NETWORK_HELPERS_STREAM_PROXY_HPP
+#define KAGOME_NETWORK_HELPERS_STREAM_PROXY_HPP
+
+#include <libp2p/connection/stream.hpp>
+
+namespace libp2p::connection {
+  struct StreamProxy : Stream {
```

Comment:
Since this proxy isn't doing anything and as far as I understand is meant only for inheriting useful proxy-classes from it, would it not be clearer to name it StreamProxyBase or something alike?